src/plugins/vis_types/timeseries/public/application/visualizations/views/timeseries/index.js

```diff
@@ -152,9 +152,6 @@ export const TimeSeries = ({
         onPointerUpdate={handleCursorUpdate}
         theme={[
           {
-            crosshair: {
```

Comment:
@nickofthyme in `src/plugins/vis_types/xy/public/components/xy_settings.tsx` we have the same code. Maybe it should be also deleted?

![image](https://user-images.githubusercontent.com/20072247/140027248-b43f6d8c-6d3f-4615-a5e3-754e74c8ea11.png)

Reply:
Thanks, done 46c74b6